code-challenges.test.js

```diff
@@ -51,17 +90,50 @@ const studyMinutesWeek2 = {
 
 // b) Create the function that makes the test pass.
 
+// PSUEDO CODE:
+// Create a function named leastToGreatest with an object as the parameter.
+// We then must use a new variable name (newArray) and use object.values to return an array of the properties own values in the same order into an array.
+// We then must sort the numbers least to greatest.
+
+const leastToGreatest = (object) => {
+  let newArray = Object.values(object).sort((a, b) => a - b)
+  return newArray
+}
+
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("accSum", () => {
+  it("takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.", () => {
+    expect(accSum(accountTransactions1)).toEqual([100, 83, 60, 51])
+    expect(accSum(accountTransactions2)).toEqual([250, 161, 261, 165])
+    expect(Accsum(accountTransactions3)).toEqual([])
+  })
+})
+
+// Good Fail.
+// ReferenceError: accSum is not defined
+
 const accountTransactions1 = [100, -17, -23, -9]
 // Expected output: [100, 83, 60, 51]
-
 const accountTransactions2 = [250, -89, 100, -96]
 // Expected output: [250, 161, 261, 165]
-
 const accountTransactions3 = []
 // Expected output: []
 
 // b) Create the function that makes the test pass.
+
+// PSEUDO CODE:
+// Create a function (accSum).
+// I need to iterate through the array and return the accumlative sum of each element
+// I'll also return the empty array since it's null
+// I have no clue how to do this and I don't want to copy anything I find
+
+const accSum = (array) => {
+  let emptyArr = []
+  if (accSum === null) {
+    console.log("[]")
+  } else if (i = 0; i < array.length; i++)
+
```

Comment:
This is a tough one.  If you'd like to go over it one on one, I'm happy to meet with you.   You were somewhat on the right track.